geps/gep-1897.md

```diff
@@ -0,0 +1,113 @@
+so in order to drive resolution this GEP focuses only on this single piece of functionality.
+
+## Goals
+
+1. The solution should satisfy the following use case: the backend pod has its own
+certificate and the gateway implementation client needs to know how to connect to the
+backend pod. (Use case #4 in [Gateway API TLS Use Cases](#references))
+2. In terms of the Gateway API personas, the application developer should control the
+gateway to backend TLS settings, not the cluster operator, as requiring a cluster operator
+to manage certificate renewals and revocations would be extremely cumbersome.
+3. The solution should consider client certificate settings used in the TLS handshake **from
+Gateway to backend**, such as TLS versions and cipher suites. (Use case #5 in [Gateway API TLS Use Cases](#references))
+
+## Non-Goals
+
+1. Changes to the existing mechanisms for edge or passthrough TLS termination
+2. Providing a mechanism to decorate multiple route instances
+3. Service mesh use cases
+4. Mutual TLS use cases
+5. TLSRoute, TCPRoute, or UDPRoute use cases
+6. Termination of TLS for HTTP routing (#1 in [Gateway API TLS Use Cases](#references))
+7. HTTPS passthrough use cases (#2 in [Gateway API TLS Use Cases](#references))
+8. Termination of TLS for non-HTTP TCP streams (#3 in [Gateway API TLS Use Cases](#references))
+9. Controlling certificates used by more than one workload (#6 in [Gateway API TLS Use Cases](#references))
+10. Client certificate settings used in the TLS handshake **from external clients to the
+Listener** (#7 in [Gateway API TLS Use Cases](#references))
+
+## Overview - what do we want to do?
+
+Given that the current ingress solution specifies **edge** TLS termination (from the edge to
+the gateway), and how to handle **passthrough** TLS (from the edge to the backend pod), this
+proposed ingress solution specifies **backend** TLS termination (from the gateway to the
+backend pod).  As mentioned, this solution satisfies the use case in which the backend pod
+has its own certificate and the gateway client needs to know how to connect to the backend pod.
+
+![image depicting TLS termination types](images/1897-TLStermtypes.png "TLS termination types")
+
+## Purpose - why do we want to this?
+
+This proposal is _very_ tightly scoped because we have tried and failed to address this well-known
+gap in the API specification. The lack of support for this fundamental concept is holding back
+Gateway API adoption by users that require a solution to the use case. One of the recurring themes
+that has held up the prior art has been interest related to service mesh and as such this proposal
+focuses only on the ingress use case to avoid contention there.  Another reason for the tight scope
+is that we have been too focused on a generic representation of everything that TLS can do, which
+covers too much ground to address in a single GEP.
+
+## Prior Art - the history of backend TLS
```

Comment:
Based on feedback from @costinm in https://github.com/kubernetes-sigs/gateway-api/pull/1906/files#r1157820085

```suggestion
## The history of backend TLS
```